### ISSUE TITLE
Break out scaffold

### ISSUE THREAD

**Issue by drGrove** [growthemes/scaffold#4]
As per conversation with @jeremydw, we should make the base scaffold as simple as possible.

We should have:
- scaffold, a vanilla js grow scaffold
- scaffold-angular, an angular based solution (basically what we have now)

**Comment by jeremydw** [comment]
Yes! We also have http://github.com/growthemes/base ("base") which is even more minimal than scaffold. Can you take a look at `base` and see if it would make sense to do:
- `base`
- `base-angular`
- `base-localized`
- `base-airkit`
- `base-appengine`
- `base-closure`
  etc.?

Note that we'd only want to create variants `-<foo>` for versions of `base` that we'd actually use.

That would be a nice way to keep things centralized around the `base` theme (which as a minimal, but basic visual design) too.
